Comanche055/CM_ENTRY_DIGITAL_AUTOPILOT.agc

```diff
@@ -992,33 +992,32 @@ TIMSCAL		TS	TOFF		# IN CS
 		CAF	4JETT
 		EXTEND
 		MP	TON1		# C(TON1) = TON1 / 4T
-		TS	TON1		# IN CS
+		TS	TON1		#  IN CS
 
 		CAF	4JETT
 		EXTEND
 		MP	TON2		# C(TON2) = TON2 / 4T
 		TS	TON2		# IN CS
 
-		CA	ZERO		# CANNOT REDO AFTER TIMETST.  TUSED GONE
+		CA	ZERO		# CANNOT REDO AFTER TIMETST. TUSED GONE
 		TS	JETAG		# SET +0 TO SHOW ROLL DAP CALLED.
 
-					# CAUSE THE TM OF BODY RATES VIA UPBUFF TO BE
-					# INITIALIZED.  ALSO CAUSE NEEDLES TO BE DONE ON EXIT
-					# AND ON ALTERNATE PASSES THROUGH CM/DUMPR.
+			     # CAUSE THE TM OF BODY RATES VIA UPBUFF TO BE
```

Comment:
It's half the width of a single tab.

On another note, you've made me notice I've accidentally hit the shortcut to convert all the tabs in this file into spaces...so i'm gonna have to redo that commit.